docs/decisions/infra/separate-terraform-backend-configs-into-separate-config-files.md

```diff
@@ -0,0 +1,12 @@
+# Notes
+
+For creating accounts, can't use the .tfbackend backend config file approach because the main.tf file can only have one backend configuration, so if we have the backend configuration as a partial configuration of `backend "s3" {}`, then we can't use that same module to configure a new account, since the process for configuring a new account
```

Comment:
For creating accounts, we can't use

Reply:
@shawnvanderjagt Sorry, the previous version of this markdown doc was just some notes that I had, not a properly written ADR that was ready for review. Anyways, I rewrote the ADR and now it's completely done. Take another look